BonApp.js

```diff
@@ -47,306 +45,305 @@ render: function(output) {
             }
-            okItems.push(item);
+            okItems.push(item)
           }
         }
       }
       if (okItems.length === 0) {
-        okItems.push('0 menu items found...');
+        okItems.push('0 menu items found...')
       }
 
-      return okItems;
-    };
+      return okItems
+    }
 
 
     var lastUpdated = function() {
-      var time = new Date;
-      var hours = time.getHours();
-      var minutes = time.getMinutes();
-      var timeOfDay = '';
+      var time = new Date
+      var hours = time.getHours()
+      var minutes = time.getMinutes()
+      var timeOfDay = ''
 
       if (hours > 12) {
-        timeOfDay = 'pm';
-        hours -= 12;
+        timeOfDay = 'pm'
+        hours -= 12
       }
       else if (hours == 12) {
-        timeOfDay = 'pm';
+        timeOfDay = 'pm'
       }
       else if (hours === 0) {
-        timeOfDay = 'am';
-        hours = 12;
+        timeOfDay = 'am'
+        hours = 12
       }
       else {
-        timeOfDay = 'am';
+        timeOfDay = 'am'
       }
 
-      return function() { return 'Updated at ' + pad(hours) + ':' + pad(minutes) + timeOfDay};
-    };
+      return function() { return 'Updated at ' + pad(hours) + ':' + pad(minutes) + timeOfDay}
+    }
 
 
     var pad = function(number) {
-      return (number < 10 ? '0' : '') + number;
-    };
+      return (number < 10 ? '0' : '') + number
+    }
 
 
     var capitalizeStr = function(theStr, upperCaseLetter, removeLetter) {
-      var newCapitalStr = theStr[upperCaseLetter].toUpperCase() + theStr.slice(removeLetter);
-      return newCapitalStr;
-    };
+      var newCapitalStr = theStr[upperCaseLetter].toUpperCase() + theStr.slice(removeLetter)
+      return newCapitalStr
+    }
 
 
     var sortByProperty = function(property) {
-      //'use strict';
+      //'use strict'
       return function(a, b) {
-        var sortStatus = 0;
+        var sortStatus = 0
         if (a[property] < b[property]) {
-          sortStatus = -1;
+          sortStatus = -1
         }
         else if (a[property] > b[property]) {
-          sortStatus = 1;
+          sortStatus = 1
         }
 
-        return sortStatus;
-      };
-    };
+        return sortStatus
+      }
+    }
 
 
     var updateTheData = (function(_this) {
       return function(cafeId) {
-        var theCommand = 'curl -s \'http://legacy.cafebonappetit.com/api/2/menus?cafe=' + cafeId + "'";
+        var theCommand = 'curl -s \'http://legacy.cafebonappetit.com/api/2/menus?cafe=' + cafeId + "'"
 
         return _this.run(theCommand, function(err, myOut) {
-          makeItAllHappen(myOut);
-          return changeTheView();
-        });
-      };
-    })(this);
+          makeItAllHappen(myOut)
+          return changeTheView()
+        })
+      }
+    })(this)
 
 
     var initSearch = function() {
-      return jQuery.get('BonApp-widget/scripts/data.json', function(data) {
+      return jQuery.get('BonApp-Widget/scripts/data.json', function(data) {
```

Comment:
Lowercase

Reply:
Sticking to uppercase for the reason stated earlier.